spk/cryptsetup/Makefile

```diff
@@ -0,0 +1,34 @@
+SPK_NAME = cryptsetup
+SPK_VERS = 2.6.1
+SPK_REV = 1
+SPK_ICON = src/papirus-folder-locked.png
+
+REQUIRE_KERNEL ?= 1
+REQUIRE_KERNEL_MODULE  = CONFIG_BLK_DEV_DM_BUILTIN:drivers/md:dm-builtin
+# MD required because of Synology patches
+REQUIRE_KERNEL_MODULE += CONFIG_BLK_DEV_MD:drivers/md:md-mod
+REQUIRE_KERNEL_MODULE += CONFIG_BLK_DEV_DM:drivers/md:dm-mod
+REQUIRE_KERNEL_MODULE += CONFIG_DM_CRYPT:drivers/md:dm-crypt
+REQUIRE_KERNEL_MODULE += CONFIG_CRYPTO_USER_API:crypto:af_alg
+REQUIRE_KERNEL_MODULE += CONFIG_CRYPTO_USER_API_HASH:crypto:algif_hash
+REQUIRE_KERNEL_MODULE += CONFIG_CRYPTO_USER_API_SKCIPHER:crypto:algif_skcipher
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = jonathan-conder
+DESCRIPTION = Open-source utility used to conveniently set up disk encryption based on the dm-crypt kernel module.
+DISPLAY_NAME = Cryptsetup
+HOMEPAGE = https://gitlab.com/cryptsetup/cryptsetup
+LICENSE  = GPLv2
+
+PLIST_TRANSFORM = sed -e 's%@kernel@%$(TC_ARCH)-$(TC_VERS)/$(TC_KERNEL)%'
```

Comment:
Instead you can simply use `lib:lib/modules/**` in the `PLIST` as there won't be any other modules directory.

Reply:
ah ok, I had another patch which changed the PLIST behaviour, but accidentally broke that behaviour. Fixed now